webpack.config.js

```diff
@@ -20,10 +20,13 @@ export default (env, { mode }) => {
     // Switch to .env files once this gets unwieldy
     const e = process.env
     const STATIC_ORIGIN = e.STATIC_ORIGIN ?? 'http://localhost:8030'
-    const L1_ORIGIN = e.L1_ORIGIN ?? 'http://localhost:8031'
-    const TRUSTED_L1_ORIGIN = e.TRUSTED_L1_ORIGIN ?? 'http://localhost:8031'
-    const UNTRUSTED_L1_ORIGIN = e.UNTRUSTED_L1_ORIGIN ?? 'https://localhost'
+    const L1_ORIGIN = e.L1_ORIGIN ?? 'https://l1s.saturn.pl'
+    const TRUSTED_L1_ORIGIN = e.TRUSTED_L1_ORIGIN ?? 'https://l1s.saturn.pl'
+    const UNTRUSTED_L1_ORIGIN = e.UNTRUSTED_L1_ORIGIN ?? 'https://l1s.saturn.pl'
```

Comment:
TLD is `.ms` not `.pl`

Reply:
updates to .ms. these are all test net urls though. Are they supposed to default to prod?

Reply:
defaulting to testnet is fine. although I don't think any of this matters since these env vars aren't being used ever since we added the js-client.

Reply:
yeah that makes sense